### PR TITLE
test: Add SDK test for all mint operations

### PR DESCRIPTION
**Pull request by RamirezAlex**
Closes: #283

### CODE REVIEW

**Comment by RamirezAlex** [comment]
@beeman the tests pass in local env. but I am not sure how to add and enable the mint in the environment that we use for CI. Other than that, it should fine for review.

**Comment by beeman** [comment]
> @beeman the tests pass in local env. but I am not sure how to add and enable the mint in the environment that we use for CI. Other than that, it should fine for review. Do you have any suggestions?

That makes sense. I think it's something that would be addressed by #285 - I'm going to pull this branch in and test if that makes it work and implement that.

File: apps/sdk-e2e/src/integration/transaction-kinetic-sdk-e2e.spec.ts [hunk 1]
```diff
@@ -157,4 +157,38 @@ describe('KineticSdk (e2e)', () => {
...
+    expect(tx.mint).toBe(usdcMint)
+    const { signature, errors, amount, source } = tx
+    expect(typeof signature).toBe('string')
+    expect(errors).toEqual([])
+    expect(Number(amount)).toBe(100)
+    expect(source).toBe(aliceKeypair.publicKey)
+  })
+
+  it('should make a batch transfer with a provided mint', async () => {
+    setTimeout(async () => {
```
Comment (beeman) [inline apps/sdk-e2e/src/integration/transaction-kinetic-sdk-e2e.spec.ts:179]: We want to get rid of this `setTimeout`

File: apps/sdk-e2e/src/integration/transaction-kinetic-sdk-e2e.spec.ts [hunk 2]
```diff
@@ -157,4 +157,38 @@ describe('KineticSdk (e2e)', () => {
...
+      await sdk.createAccount({ owner: bobKeypair, mint: usdcMint })
+      const destinations: Destination[] = [{ destination: bobKeypair.publicKey, amount: '2' }]
+
+      const tx = await sdk.makeTransferBatch({ destinations, owner: aliceKeypair, mint: usdcMint })
+      console.dir(tx, { depth: null })
```
Comment (beeman) [inline apps/sdk-e2e/src/integration/transaction-kinetic-sdk-e2e.spec.ts:184]: Rogue logging

**Review by beeman — CHANGES_REQUESTED** [review]
@RamirezAlex two small comments on this one.

**Comment by beeman** [comment]
@RamirezAlex make sure to pull in the branch before working on it. I added a way to run the sdk-e2e against the local API (without spinning up its own) which makes debugging easier.

```
yarn nx run sdk-e2e:e2e-local --verbose --watch
```

**Review by beeman — APPROVED** [review]

**Pull request merged**